internal/testutils/utils.go

```diff
@@ -3,3 +3,11 @@ package testutils
 func IntPtr(i int) *int {
 	return &i
 }
+
+func StrPtr(s string) *string {
```

Comment:
How about putting them to `utils` packages instead of `testutils`? 
or there "k8s.io/utils/pointer" with `StringPtr` and `BoolPtr`

Reply:
Moved to `utils/conversions` (utils had an import loop) in commit 6e71482. It affected a lot of files.